xml/System.Net.Sockets/Socket.xml

```diff
@@ -6281,7 +6281,11 @@ Duplication of the socket reference failed.</exception>
 
  ]]></format>
         </remarks>
-        <exception cref="T:System.NotSupportedException">Windows NT is required for this method.</exception>
```

Comment:
The Windows 95-Me product line has been out of support for a long time. Dow we still need this line? (Same applies for `EndAccept()`.)